config/webpack.config.dev.js

```diff
@@ -68,21 +112,22 @@ module.exports = ((webpackEnv) => {
       publicPath: publicPath,
       // Prevents conflicts when multiple webpack runtimes (from different apps)
       // are used on the same page.
-      jsonpFunction: `webpackJsonp${appPackageJson.name}`,
+      jsonpFunction: `webpackJsonp${require(paths.appPackageJson).name}`,
       // this defaults to 'window', but by setting it to 'this' then
       // module chunks which are built will work in web workers as well.
       globalObject: 'this',
     },
 
     optimization: {
+      moduleIds: 'named',
```

Comment:
We could also use this property in prod config instead of `webpack.HashedModuleIdsPlugin`. Check all the options:
https://webpack.js.org/configuration/optimization/#optimizationmoduleids